### ISSUE TITLE
add BatchNormalization layers after 1 * 1 convolution in the shortcut path

### ISSUE THREAD

**Issue by sjtudesigner** [raghakot/keras-resnet#12]
In the official version keras.datasets.resnet50, the 1 * 1 convolution layer on the shortcut path is continued with a batchnormalization layer. I tried the network without this layer with CIFAR-10 dataset, and it seems the response of convolution path is overwhelmed by the shortcut path. In this way, training has to be very slow, then reach a low accuracy plateau.

**Comment by raghakot** [comment]
It's definitely possible. The intent of this module was to provide a quick way to build your own resnets with the resnet-50 as a sample. By no means is that a fixed architecture.
